src/OSPSuite.UI/Binders/AxisBinder.cs

```diff
@@ -163,10 +163,14 @@ private void adjustAxisMinMax()
          else // somebody inserted a limit...
          {
             if (!Axis.Min.HasValue)
-               Axis.Min = Convert.ToSingle(_axisView.WholeRange.MinValue);
+            {
```

Comment:
Here it was possible that user sets an explicit value on one property and the other property automatically calculates an inconsistent value (eg. min > max).